src/proto/beats.proto

```diff
@@ -10,10 +11,23 @@ enum Source {
   USER = 2;
 }
 
+enum Type {
+  BEAT = 0;
+  BAR = 1;
+  PHRASE = 2;
+}
+
+message Signature {
+  required int32 beats = 1;
```

Comment:
I presume this means beats per bar? Simply calling it "beats" is unclear.

Reply:
Property name changed for clarity.